templates/graph.tpl

```diff
@@ -0,0 +1,250 @@
+{
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "CloudWatch",
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 9,
+        "w": 12,
+        "x": 0,
+        "y": 0
+      },
+      "id": ${graph_id},
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "nullPointMode": "null",
+      "options": {
+        "dataLinks": []
+      },
+      "percentage": false,
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "dimensions": {
+            "BucketName": "${bucket_name}",
+            "StorageType": "AllStorageTypes"
+          },
+          "expression": "",
+          "highResolution": false,
+          "id": "",
+          "metricName": "NumberOfObjects",
+          "namespace": "AWS/S3",
+          "period": "300",
+          "refId": "A",
+          "region": "us-east-1",
```

Comment:
can we use `data.aws_region.current.name` to configure this instead of hardcoding?

Reply:
👍  - I did this in a previous version and must have overwritten it at some point.